### PR TITLE
calculator 

### PR DESCRIPTION
**Pull request by Bhavani-Shankar13**
This program is a Simple Calculator implemented in Java. It allows users to perform basic arithmetic operations such as addition, subtraction, multiplication, and division. 
The program runs in a loop, continuously prompting the user for input until they choose to exit by entering X or x.

### CODE REVIEW

**Review by lijojosef — COMMENTED** [review]
Can you move these files in a proper folder structure? 
eg: `Bhavani-Shankar-practice/Calculator.java`

File: com/lijo/basics/calculator.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import java.util.*;
+public class calculator {
```
Comment (lijojosef) [inline com/lijo/basics/calculator.java:2]: Class Name should always start with a capital letter and also should match the file name
`Calculator.java`

File: com/lijo/basics/calculator.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            if (input.equalsIgnoreCase("X")) {
+                System.out.println("Exiting...... !");
+                break;
+            }
+            System.out.println("Enter the two numbers for the claculation :");
+
+
+            int number1 = in.nextInt();
+            int number2 = in.nextInt();  
+            in.nextLine();
```
Comment (lijojosef) [inline com/lijo/basics/calculator.java:19]: Is this line required?

File: com/lijo/basics/calculator.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                case "4":
+                    if (number2 == 0) {
+                        System.out.println("Division by zero is Undefined.");
+                        continue;
+                    }
+                    result = (number1 / number2);
+                    break;
+                default:
+                    System.out.println("Invalid operator.");
+                    continue;
```
Comment (lijojosef) [inline com/lijo/basics/calculator.java:40]: Is this `continue` required?

File: com/lijo/basics/calculator.java (hunk 1 above)
Comment (Bhavani-Shankar13) [reply com/lijo/basics/calculator.java:2]: okay sir

File: com/lijo/basics/calculator.java (hunk 3 above)
Comment (Bhavani-Shankar13) [reply com/lijo/basics/calculator.java:40]: No sir ,without continue keyword  it also works same way.

**Pull request closed without merging**